prow/config.yaml

```diff
@@ -6107,6 +6107,22 @@ periodics:
       - "--repo=k8s.io/autoscaler=master"
       - "--root=/go/src"
       - "--timeout=60"
+      env:
+      - name: DOCKER_IN_DOCKER_ENABLED
+        value: true
+      # docker-in-docker needs privileged mode
+      securityContext:
+        privileged: true
+      volumeMounts:
+      - name: docker-graph
+        mountPath: /docker-graph
+      # port set to limit to one job per node since sharing docker isn't safe
+      ports:
```

Comment:
since you have emptyDir please drop the port. to clarify: sharing the directory isn't safe. multiple d-in-d jobs with *different* directories is safe, which emptyDir gets us.

Reply:
Getting rid of the port definition resulted in failed check: https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/test-infra/6817/pull-test-infra-bazel/13437/

Reply:
I will fix this in a follow up :(
Proliferating these jobs is hurting presubmit throughput :/